src/state/upload/upload-reducers.js

```diff
@@ -37,9 +38,24 @@ export default function (state = initialState, action) {
 
         case uploadActionTypes.FETCH_ITEMS_SUCCEEDED: {
             const { groups } = action.data;
+            let ID = null;
+            for (let i = 0; i < groups.length; i += 1) {
+                if (groups[i].name === 'Generic') {
+                    ID = groups[i].id;
+                }
+            }
+            let array = [];
+            array = array.concat(ID);
             return {
                 ...state,
-                groups: groups.map((object) => { const obj = object; obj.checkbox = false; return obj; }),
+                groups: groups.map((object) => {
```

Comment:
it's named "groups" because that's what the JSON object in the backend is named...so it'll be consistent